emit_translations.py

```diff
@@ -84,6 +85,8 @@ def main():
                     help='Output directory for translated source files.')
     ap.add_argument('-v', '--verbose', action='store_true',
                     help='Enable verbose logging')
+    ap.add_argument('--output-csv', type=str, required=False,
```

Comment:
```suggestion
    ap.add_argument('--output-csv', type=pathlib.Path, required=False,
```

Recently I found out that the `type` argument can be any callable that accepts a string (e.g., a constructor); so it may be a bit more idiomatic to use `pathlib.Path` to specify the intent of this argument (also gives you immediate access to that class's methods which could be useful later).

Reply:
Good call, changed path args in emit_translations.py to use this